modules/Twitter/src/main/java/totetmatt/gephi/twitter/CredentialProperty.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Consider deleting this boilerplate code or changing it to something more meaningful:
```To change this license header....```